### PR TITLE
Use Dropout Layer in OpenAIGPTMultipleChoiceHead

### LINKED ISSUE #354: Dropout Layer in OpenAIGPTMultipleChoiceHead not used
[OpenAIGPTMultipleChoiceHead](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/modeling_openai.py#L363) defines an additional dropout layer, which is not used in `forward`.

### PR DESCRIPTION
**Pull request by pglock**
closes #354

### CODE REVIEW

**Comment by thomwolf** [comment]
Seems good to me, thanks for that.
Let me just check why we don't have Circle-CI tests on the PR anymore and I'll merge it.
